property/index.js

```diff
@@ -70,6 +71,7 @@ module.exports = yeoman.generators.Base.extend({
         name: 'name',
         message: 'Enter the property name:',
         validate: checkPropertyName,
+        default: this.propDefinition && this.propDefinition.name,
```

Comment:
Somehow this does not seem to work for me.

```
$ yo loopback:property
? Select the model: Avatar
? Enter the property name: num
? Property type: array
? The type of array items: number
? Required? Yes
? Default value[leave blank for none]: asd
Warning: please enter the num property again. The default value provided "asd" is not valid for the selected type: number
? Enter the property name:
```

No default was offered for the property name. PTAL